### PR TITLE
I18n complement

### PR DESCRIPTION
**Pull request by biharck**
Adding missing i18n properties

### CODE REVIEW

File: src/components/studyList/StudyList.js [hunk 1]
```diff
@@ -256,28 +257,28 @@ class StudyList extends Component {
...
         sort: 0,
       },
       patientId: {
-        displayText: 'MRN',
+        displayText: this.props.t('MRN'),
         sort: 0,
       },
       accessionNumber: {
-        displayText: 'Accession #',
+        displayText: this.props.t('Accession'),
```
Comment (swederik) [inline src/components/studyList/StudyList.js]: Can we make the key for this AccessionNumber instead? That will make it more clear for translators what this is for.

File: src/components/studyList/StudyList.js [hunk 2]
```diff
@@ -256,28 +257,28 @@ class StudyList extends Component {
...
         sort: 0,
       },
       modalities: {
-        displayText: 'Modality',
+        displayText: this.props.t('Modality'),
         sort: 0,
       },
       studyDescription: {
-        displayText: 'Description',
+        displayText: this.props.t('Description'),
```
Comment (swederik) [inline src/components/studyList/StudyList.js]: Can we make the key for this StudyDescription?

File: src/components/measurementTable/MeasurementTable.js [hunk 3]
```diff
@@ -149,7 +149,7 @@ class MeasurementTable extends Component {
     return this.props.timepoints.map((timepoint, index) => {
       return (
         <div key={index} className="measurementTableHeaderItem">
-          <div className="timepointLabel">{timepoint.label}</div>
+          <div className="timepointLabel">{this.props.t(timepoint.key)}</div>
```
Comment (swederik) [inline src/components/measurementTable/MeasurementTable.js:152]: Why are you changing this from `timepoint.label` to `timepoint.key`?

Key is going to be something like baseline, current, comparison, or nadir. We want to display the label or the timepoint, e.g. Baseline, Follow-up 1, Follow-up 2, ..., not the key.

(though I don't know how this is going to show up in Locize...)

**Comment by swederik** [comment]
What about this one? https://github.com/OHIF/react-viewerbase/blob/873685a1c00dff577fd1c6e348c6fd707f5df533/src/components/layoutButton/LayoutButton.js#L49

File: src/components/studyList/StudyList.js (hunk 2 above)
Comment (biharck) [reply src/components/studyList/StudyList.js]: Done

File: src/components/studyList/StudyList.js (hunk 1 above)
Comment (biharck) [reply src/components/studyList/StudyList.js]: Done

**Pull request merged**